Flatpak/firststart/ui/assistant.glade

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.40.0 -->
+<interface>
+  <requires lib="gtk+" version="3.12"/>
+  <object class="GtkTextBuffer" id="textbuffer_end">
+    <property name="text" translatable="yes">In case you want to change your startup directory later,
+you can launch this wizard from the terminal:
+
+flatpak run --command=easyrpg-firststart org.easyrpg.player</property>
+  </object>
+  <object class="GtkTextBuffer" id="textbuffer_start">
+    <property name="text" translatable="yes">EasyRPG Player will usually start from the current
+working directory. However, this is not too useful
+in a Flatpak environment.
+
+Therefore this wizard lets you choose a different
+startup folder.
+
+Navigate to your game directory
+(e.g. /home/user/Downloads/RPG) and save.</property>
```

Comment:
Imo the text should be more user agnostic and not explain the technical reason, e.g.:

> This wizard lets you choose the game directory of EasyRPG Player.
> 
> Navigate to your game directory containing RPG Maker 2000/2003 games
(e.g. /home/user/Downloads/RPG) and save.